### PR TITLE
Add clarifying login text and fill dashboard lorem ipsum

### LINKED ISSUE #144: Put real text in dashboard API section
Currently lorem ipsum. Not awesome. The section doesn't per se have to be copy about the API. We can even ixnay it but I think the dashboard could benefit from some text about the lab, as its de facto homepage.

### PR DESCRIPTION
**Pull request by fungjj92**
### Demo

screen shot 2017-07-12 at 4 21 08 pm
screen shot 2017-07-12 at 3 28 39 pm

### Notes

I tweaked the suggested text to fit the space better

## Testing Instructions
`npm run serve`

Closes #144  #155

### CODE REVIEW

File: src/app/login/login.component.html [hunk 1]
```diff
@@ -5,6 +5,8 @@
     <div class="login-body">
         <div class="login-box">
             <h3>Login</h3>
+            <br>Using your Climate API account information
```
Comment (CloudNiner) [inline src/app/login/login.component.html]: Can probably drop the word "information"?

**Comment by CloudNiner** [comment]
Check Jenkins as well?

**Comment by fungjj92** [comment]
I am not alone, the internet is buzzing w this problem today: https://stackoverflow.com/questions/45077614/cannot-create-property-creationtrace-on-string-zone-symbol-optimizedzo

File: src/app/dashboard/dashboard.component.html [hunk 2]
```diff
@@ -50,7 +50,11 @@ <h2 class="title margin-no">Lab</h2>
                 <h2>API</h2>
             </div>
             <div class="about-body">
-                Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum
+                We created Azavea's Climate API in order to give you easy access to the NEX-GDDP data without the hassle of having to download and wrangle a NetCDF dataset. With each query, you can choose from over 1,000 U.S. locations, either of the two most commonly used RCPs, 22 indicators of temperature and precipitation, and anytime 1950-2100.
```
Comment (jcahail) [inline src/app/dashboard/dashboard.component.html]: sounds good.

File: src/app/login/login.component.html (hunk 1 above)
Comment (jcahail) [inline src/app/login/login.component.html]: I'd recommend leaving it--the information in this case refers to email and password; if you just say your account, people might question if they're logging into the right place.

Comment (fungjj92) [inline src/app/login/login.component.html]: ^ my thought process

**Comment by fungjj92** [comment]
All updated and build fixed 🎉

**Pull request merged**